### PR TITLE
Persist values on back in create batch

### PR DESCRIPTION
**Pull request by randi274**
# Critical Changes

# Changes
- Persist values on back button click. Prevent user from moving forward or back if errors are present in a step.
# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000007fPhtQAE" target="_blank">W-029930</a>

File: src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardController.js [hunk 1]
```diff
@@ -32,51 +32,69 @@
...
                 possibleError = component.get('v.templateMetadata.labels.missingNameDescriptionError');
-                model.getTemplateMetadata().nextStep(isValid, possibleError);
             } else if (step === '2') {
                 //handle template selection and copying here
             } else if (step === '3') {
-                model.getTemplateFields().updateToActive(component.get('v.templateFields').fieldGroups);
+                // model.getTemplateFields().updateToActive(component.get('v.templateFields').fieldGroups);
                 possibleError = model.getTemplateFields().getRequiredFieldErrors();
                 isValid = (possibleError.length === 0);
-                model.getTemplateMetadata().nextStep(isValid, possibleError);
```
Comment (lmeerkatz) [inline src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardController.js]: What does removing this do?

File: src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardController.js [hunk 2]
```diff
@@ -32,51 +32,69 @@
...
             if (step === '1') {
                 model.getTemplateInfo().load(component.get('v.templateInfo'));
                 isValid = model.getTemplateInfo().isValid();
                 possibleError = component.get('v.templateMetadata.labels.missingNameDescriptionError');
-                model.getTemplateMetadata().nextStep(isValid, possibleError);
             } else if (step === '2') {
                 //handle template selection and copying here
             } else if (step === '3') {
-                model.getTemplateFields().updateToActive(component.get('v.templateFields').fieldGroups);
+                // model.getTemplateFields().updateToActive(component.get('v.templateFields').fieldGroups);
```
Comment (lmeerkatz) [inline src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardController.js]: Delete this line if we don't need it

File: src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardController.js (hunk 1 above)
Comment (lmeerkatz) [reply src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardController.js]: It looks like the arguments being passed in here weren't doing anything - https://github.com/SalesforceFoundation/Cumulus/blob/1b7bbc23f9f21a060355324970fe63deaa149d10/src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardHelper.js#L797

What's that about?

**Pull request merged**